crates/sparrow-runtime/src/execute/operation/shift_to.rs

```diff
@@ -333,9 +333,7 @@ impl ShiftToColumnOperation {
             shifted_times_primitive,
         )?;
 
-        let num_filtered = is_forward_shift.len()
-            - is_forward_shift.data().buffers()[0]
-                .count_set_bits_offset(is_forward_shift.offset(), is_forward_shift.len());
+        let num_filtered = is_forward_shift.len() - is_forward_shift.values().count_set_bits();
```

Comment:
Does this not need the  `offset` anymore?

Reply:
Nope -- the new version of arrow returns a `NullBuffer` which has an internal offset and length. This is part of making these operations less error prone and easier for us to work with.